apiproxy/resources/jsc/set-response.js

```diff
@@ -27,7 +27,7 @@
     jws.token_type   = "Bearer";
 
      // for /token flow
-     jws.expires_in = context.getVariable("oauthv2accesstoken.AccessTokenRequest.expires_in");
+     jws.expires_in = parseInt(context.getVariable("oauthv2accesstoken.AccessTokenRequest.expires_in"));
 
      // for any other flows if any 
     if ( !jws.expires_in ) {

```

Comment:
Null handling will fail in this code. It needs to be implemented as an else case.